docs/download/downloading_data.md

```diff
@@ -0,0 +1,261 @@
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#gdc)
+
+#### Prerequisites
+
+1. Obtain dbGaP access and an NIH eRA Commons account (refer to instructions [here](https://gdc.cancer.gov/access-data/obtaining-access-controlled-data))
+2. Download the [GDC Data Transfer Tool](https://gdc.cancer.gov/access-data/gdc-data-transfer-tool)
+3. Once you have obtained controlled data access from dbGaP, you will need to download your access authentication token from the GDC Data Portal. Refer to GDC documentation [here](https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload/#obtaining-an-authentication-token-for-data-downloads) for instructions. 
+4. Save the GDC authentication token in a text file (example: `gdc-user-token.txt`)
+
+#### Download Data Instructions
+
+1. Find your data of interest in the [Data Repository](https://dcc.icgc.org/repositories) by selecting `GDC` repository. Click on `Download Files` and download the manifest file
+
+    ![Download-GDC-Manifest](images/download-gdc-manifest.png)
+
+2. Run the GDC Client tool to download data
+
+```
+./gdc-client download -m test/manifest.gdc.1554495935637.tsv -t gdc-user-token.txt -d gdc_downloads
+```
+
+## Downloading data from EGA repository
+
+The European Genome-Phenome Archive ([EGA](https://ega-archive.org/)) is co-managed by [EBI](https://www.ebi.ac.uk/) and [CRG](http://www.crg.eu/). Data can only be downloaded through their EGA download client, but metadata may be viewed on their website. Files are grouped into datasets based on the study they were collected in, and access is granted on a dataset by dataset basis. This repository carries both ICGC and non-ICGC data.
+
+<span style="color:blue">NOTE: The "EGA - Hinxton" repository in the [Data Portal](https://dcc.icgc.org/repositories?filters=%7B%22file%22:%7B%22repoName%22:%7B%22is%22:%5B%22EGA%20-%20Hinxton%22%5D%7D%7D%7D&files=%7B%22from%22:1%7D) only contains PCAWG data. You can download other non-PCAWG ICGC data directly from [EGA](https://ega-archive.org/) (instructions on their website using their [EGA Download Client](https://ega-archive.org/download/using-ega-download-client) or [API](https://ega-archive.org/download/using-ega-api))</span>
+
+Repository information can be found [here](https://docs.icgc.org/download/repositories/#ega)
+
+#### Prerequisites
+
+1. [Apply for DACO access](https://github.com/icgc-dcc/dcc-docs/blob/download-doc-update/docs/download/data-access.md#apply-for-access-to-controlled-data). Once you are approved by DACO, you will recieve an email from EGA about setting up your password. If you already had an EGA account from before, you will use the same username/password to access ICGC controlled data at EGA.
```

Comment:
relative URL, `[Apply for DACO access](/download/data-access/#apply-for-access-to-controlled-data)`

Reply:
added relative URL